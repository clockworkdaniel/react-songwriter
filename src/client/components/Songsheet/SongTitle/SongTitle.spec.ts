// import * as React from "react";
// import { shallow } from "enzyme";
// import SongTitle from "./SongTitle";

// describe("SongTitle", () => {
//   const rename = jest.fn();

//   const wrapper = shallow(<SongTitle title="Dear Prudence" rename={rename} />);

//   it("renders as expected", () => {
//     expect(wrapper).toMatchSnapshot();
//   });
//   it("calls the rename function prop when the button is clicked", () => {
//     wrapper.find(".ss-title__rename-button").simulate("click");
//     expect(rename.mock.calls.length).toEqual(1);
//   });
// });
