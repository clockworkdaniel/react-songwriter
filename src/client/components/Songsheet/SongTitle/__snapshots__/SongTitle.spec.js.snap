// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongTitle renders as expected 1`] = `
<div
  className="songsheet__title ss-title"
>
  <h1
    className="ss-title__heading controls__container"
  >
    Dear Prudence
    <span
      className="controls"
    >
      <button
        className="controls__edit ss-title__rename-button"
        onClick={[Function]}
        type="button"
      >
        <span
          className="icon-pencil"
        />
      </button>
    </span>
  </h1>
</div>
`;
